documentation/__README.md

```diff
@@ -0,0 +1,25 @@
+## Guidelines for maintaining documentation.
+
+The `./documentation` folder is the root documentation folder for this repository.
+
+Within this documentation folder is a clone of the repo's [**wiki**](https://github.com/openmsupply/application-manager-server/wiki) repository, which appears as a submodule in the main repo, in `./documentation/_wiki`
```

Comment:
Yeah, but just remember when published to the wiki the URLs "flatten" the folder structure --  i.e. they'll appear with the URL `/wiki/<Page-name>` regardless of folder nesting.

I might not bother separating them in the back end, since all the "public" docs should all go together anyway -- so let's keep them in the front-end for now?